stories/components/agenda.stories.js

```diff
@@ -6,12 +6,7 @@
  */
 
 import { html } from 'lit-element';
-import { withA11y } from '@storybook/addon-a11y';
-import { withKnobs } from '@storybook/addon-knobs';
-import { withWebComponentsKnobs } from 'storybook-addon-web-components-knobs';
-import { withSignIn } from '../../.storybook/addons/signInAddon/signInAddon';
 import { withCodeEditor } from '../../.storybook/addons/codeEditorAddon/codeAddon';
-import '../../packages/mgt-components/dist/es6/components/mgt-agenda/mgt-agenda';
 
 export default {
   title: 'Components | mgt-agenda',
```

Comment:
I can only add a comment on this line for some odd reason. Can you remove the parameters options config too in these stories?

Reply:
Great idea, done!